conjure-undertow-processor/src/main/java/com/palantir/conjure/java/undertow/processor/data/ParamTypesResolver.java

```diff
@@ -115,8 +115,9 @@ public ParamTypesResolver(ResolverContext context) {
         throw new SafeIllegalStateException("Not possible");
     }
 
-    private ParameterType bodyParameter(EndpointName endpointName, AnnotationReflector annotationReflector) {
-        String deserializerName = InstanceVariables.joinCamelCase(endpointName.get(), "Deserializer");
+    private ParameterType bodyParameter(VariableElement variableElement, AnnotationReflector annotationReflector) {
+        String javaParameterName = variableElement.getSimpleName().toString();
+        String deserializerName = InstanceVariables.joinCamelCase(javaParameterName, "Deserializer");
```

Comment:
This is the only real change, the rest is unwinding the unused `EndpointName` var.

This way we name body deserializers using the same strategy as other parameter types (also solving a potential collision if a parameter name matches the endpoint name!)